node-js-basics-101/node_modules/engine.io-client/lib/transports/polling-xhr.js

```diff
@@ -0,0 +1,413 @@
+/**
+ * Module requirements.
+ */
+
+var XMLHttpRequest = require('xmlhttprequest-ssl');
```

Comment:
__[jshint]__
* Redefinition of 'XMLHttpRequest'.

<!-- Comment by [SideCI](https://sideci.com) -->